apps/factory/src/validators/prompt_version.ts

```diff
@@ -39,7 +39,14 @@ export const createVersionInput = z
   .object({
     promptPackageId: z.string(),
     promptTemplateId: z.string(),
-    version: z.string(),
+    version: z
+      .string()
+      .refine((version) => version.trim() !== "", {
+        message: "Version must not be empty",
+      })
+      .refine((version) => /^\d+\.\d+\.\d+$/.test(version), {
```

Comment:
@ashwaniMaddheshiya semver package is added in the codebase, please use that instead of writing custom checks. Check of semver in the codebase